Core/GDCore/Extensions/Metadata/ValueTypeMetadata.h

```diff
@@ -178,6 +178,15 @@ class GD_CORE_API ValueTypeMetadata {
     } else if (type == "variable") {
       return parameterType == "objectvar" || parameterType == "globalvar" ||
              parameterType == "scenevar";
+    } else if (type == "resource") {
+      return parameterType == "fontResource" ||
+             parameterType == "soundfile" ||
```

Comment:
just a random question, why are those types not CamelCase?

Reply:
It's legacy. There is a todo somewhere to renamed them "audioResource".